src/ui/public/directives/paginate.js

```diff
@@ -40,6 +40,12 @@ uiModules.get('kibana')
     },
     controllerAs: 'paginate',
     controller: function ($scope, $document) {
+      // Unfortunately this controller relies on inherited scope. Lets not fail if onPageChanged happens to not be
+      // implemented.
+      if (!$scope.onPageChanged) {
```

Comment:
Actually I don't think it would matter either way.  If the parent scope set the function after we set it to noop here, wouldn't it just be overwritten to now be the right function?  I don't actually see a difference one way or the other, but I'm fine with switching to checks too.

... well, if the parent explicitly deleted the function after we made it a noop, then it would fail.  And these are the reasons I don't like inherited scope.  Well, hopefully this will get moved over to the new pager soon anyway.